src/vscode.api.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  DecorationOptions,
+  Range,
+  TextEditor,
+  TextEditorDecorationType,
+  window,
+} from "vscode";
+
+// eslint-disable-next-line import/prefer-default-export
+export const setDecoration = (
+  decorationType: TextEditorDecorationType,
+  rangesOrOptions: Range[] | DecorationOptions[],
+  editor?: TextEditor
+): void =>
+  (editor ?? window.activeTextEditor)?.setDecorations(
```

Comment:
This is the hack you implemented, @dimacodota, but I made a small change to it. 
Consider the following situation: we display a decorator in one editor tab, while it's displayed the user switches tabs. Now we wish to clear the decorator. 
If we don't take the editor as a parameter and just use the currently active editor tab, then we will clear the decorator on the currently active tab and not on the tab where it was actually displayed.